backend/spec/requests/books_spec.rb

```diff
@@ -0,0 +1,33 @@
+# RSpec test for Books Controller
+require 'rails_helper'
+
+RSpec.describe 'Books', type: :request do
+  describe 'POST /books/' do
+    it 'returns books object if params are valid' do
+      post('/books/', params: {
+             book: {
+               title: 'Angels and Demons',
+               description: 'Angels & Demons is a 2000 bestselling mystery-thriller novel'
+             }
+           })
+      expect(response.status).to eql(201)
+      expect(json_body[:data]).to include({
+                                            title: 'Angels and Demons',
+                                            description: 'Angels & Demons is a 2000 bestselling mystery-thriller novel'
+                                          })
+      expect(json_body[:errors]).to eql(nil)
+    end
+
+    it 'returns error if there is a missing param' do
+      post('/books/', params: {
+             book: {
+               unknown: 'Unknown'
+             }
+           })
+      expect(response.status).to eql(406)
```

Comment:
`nit`

I think `422` is a better fit than `406` error code since the request is correct, but there are validation errors that prevent the server from processing it correctly